jsonapi_requests/orm/api_model.py

```diff
@@ -148,6 +149,7 @@ def as_identifier(self):
     def refresh(self):
         api_response = self.endpoint.get()
         jsonapi_response = api_response.content
+        assert self.id
```

Comment:
Looks like this should be done before the request is made, not after.
Maybe a dedicated nicely-named exception would be better than an assertion?

Reply:
done